src/components/Task.js

```diff
@@ -1,63 +1,97 @@
-import React from "react";
-import Button from "./Button";
-import Input from "./InputField";
-import "./Task.css";
+import React from 'react';
 
+import Button from './Button';
+import Input from './InputField';
+import classNames from 'classnames';
+
+/**
+  * This is the main Class.
+  */
 class Task extends React.Component {
-  constructor() {
+ 
+  /**
+   * @param {object} props
+   * @memberof Task
+   */
+  constructor(props) {
     super();
+
     this.state = {
+      editedContent: props.todo.value,
       edit: false
     };
   }
-
   render() {
+    const spanClass =classNames('tasks', {
+      completed:this.props.todo.done
+    });
+
+    var buttonClass = classNames({
+      'undo':this.props.todo.done,
+      'complete':!this.props.todo.done
+    })
     if (!this.state.edit) {
       return (
-        <div className='task clearfix'>
-          <span className={this.props.todo.done ? "tasks complete" : "tasks"}>
+        <li className='task clearfix'>
+          <span className={spanClass}>
             {this.props.todo.value}
           </span>
           <div class='buttons'>
             <button
-              className={this.props.todo.done ? "Undo" : "Complete"}
-              onClick={() => this.props.handleComplete(this.props.index)}
+              className={buttonClass }
```

Comment:
remove extra unnecessary spaces.

```suggestion
              className={buttonClass}
```